src/engine/components/SkyBox.cpp

```diff
@@ -57,10 +57,11 @@ namespace lei3d
 			}
 		}
 		GLCall(glTexParameteri(GL_TEXTURE_CUBE_MAP, GL_TEXTURE_MAG_FILTER, GL_LINEAR));
-		GLCall(glTexParameteri(GL_TEXTURE_CUBE_MAP, GL_TEXTURE_MIN_FILTER, GL_LINEAR));
+		GLCall(glTexParameteri(GL_TEXTURE_CUBE_MAP, GL_TEXTURE_MIN_FILTER, GL_LINEAR_MIPMAP_NEAREST));
```

Comment:
Is this all we need to do with OpenGL to support Mipmaps?

Reply:
`glGenerateMipmap()` does the other part of the work, otherwise we'll get incomplete texture errors in places. We generate mipmaps once we have the texture written to, so it's in the early stage of SSR.